### PR TITLE
[promptfile] fixed

### PR DESCRIPTION
**Pull request by OndrejIT**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

File: youtube_dl/extractor/promptfile.py [hunk 1]
```diff
@@ -33,7 +33,12 @@ def _real_extract(self, url):
             raise ExtractorError('Video %s does not exist' % video_id,
                                  expected=True)
 
+        chash_pattern = r'\$\("#chash"\)\.val\("(.+)"\+\$\("#chash"\)'
+        chash = self._search_regex(chash_pattern, webpage, "chash")
         fields = self._hidden_inputs(webpage)
+        k = fields.keys()[0]
```
Comment (dstftw) [inline youtube_dl/extractor/promptfile.py]: `keys()` is not a list under python 3. There is also no guarantee `chash` is the only in `fields` dict.

**Comment by dstftw** [comment]
Does not works for me at all:
```
> py -2.7 .\youtube_dl\__main__.py http://www.promptfile.com/l/86D1CE8462-576CAAE416 -v
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'http://www.promptfile.com/l/86D1CE8462-576CAAE416', u'-v']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2016.09.24
[debug] Git HEAD: 7fd8870
[debug] Python version 2.7.12 - Windows-10-10.0.10240
[debug] exe versions: ffmpeg N-81192-g04da20e, ffprobe N-81192-g04da20e, rtmpdump 2.4
[debug] Proxy map: {}
[PromptFile] 86D1CE8462-576CAAE416: Downloading webpage
[PromptFile] 86D1CE8462-576CAAE416: Downloading video page
ERROR: Unable to extract URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\dev\youtube-dl\master\youtube_dl\YoutubeDL.py", line 694, in extract_info
    ie_result = ie.extract(url)
  File "C:\dev\youtube-dl\master\youtube_dl\extractor\common.py", line 355, in extract
    return self._real_extract(url)
  File "C:\dev\youtube-dl\master\youtube_dl\extractor\promptfile.py", line 48, in _real_extract
    url = self._html_search_regex(r'url:\s*\'([^\']+)\'', webpage, 'URL')
  File "C:\dev\youtube-dl\master\youtube_dl\extractor\common.py", line 667, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "C:\dev\youtube-dl\master\youtube_dl\extractor\common.py", line 658, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

File: youtube_dl/extractor/promptfile.py (hunk 1 above)
Comment (OndrejIT) [inline youtube_dl/extractor/promptfile.py]: ok thx

**Comment by OndrejIT** [comment]
Fixed, sorry...

python test/test_download.py TestDownload.test_PromptFile
[PromptFile] 86D1CE8462-576CAAE416: Downloading webpage
[PromptFile] 86D1CE8462-576CAAE416: Downloading video page
[info] Writing video description metadata as JSON to: 86D1CE8462-576CAAE416.info.json
[debug] Invoking downloader on 'http://www.promptfile.com/file/eyJpIjoiODZEMUNFODQ2Mi01NzZDQUFFNDE2IiwiZSI6MTQ3NDgyMTkwMCwidCI6InN0cmVhbSIsImgiOiI5YjAxMWYzNWI2NDA2YTc4YWRjNWFmOWQwMmM0MWU4NmYzOGM3ODc5IiwicyI6MX0='
[download] Destination: 86D1CE8462-576CAAE416.mp4
[download] 100% of 10.00KiB in 00:00
.
----------------------------------------------------------------------
Ran 1 test in 0.897s

OK

**Comment by yan12125** [comment]
Not merged to master: https://github.com/rg3/youtube-dl/commits/master/youtube_dl/extractor/promptfile.py. Seems it's Github bug again. @OndrejIT Could you open a new pull request?

**Pull request merged**